hbotrc/__init__.py

```diff
@@ -0,0 +1,8 @@
+__version__ = "0.1.0"
```

Comment:
I would prefer to install it as a package in environment_conda.yml
```
name: dashboard
channels:
  - conda-forge
dependencies:
  - python=3.9
  - sqlalchemy
  - pip
  - pip:
      - ccxt
      - streamlit
      - watchdog
      - plotly
      - pycoingecko
      - glom
      - defillama
      - statsmodels
      - pandas_ta
      - pyyaml
      - commlib-py
      - git+https://github.com/hummingbot/hbot-remote-client-py.git

```

Reply:
yes I agree, but in this initial draft I made some quick changes to the code and I was testing how to run async tasks in streamlit but definitely we need to use the same repo.

Reply:
perfect! Tomorrow we have the first live session. I think that @fengtality is going to create some tasks around how we can start working together on this.
from my side I will clean up this branch and probably we can merge it so people can start using it and then we can split our work in different pages